knowledge-base/map-change-marker-colors.md

```diff
@@ -0,0 +1,130 @@
+---
+title: How to Change Map Markers Colors
+description: Learn how to customize the appearance of map markers by setting and changing their colors in a Blazor application.
+type: how-to
+page_title: How to Change Map Markers Colors
+slug: map-kb-change-marker-colors
+tags: map, markers
+res_type: kb
+ticketid: 1675518
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Map for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+When integrating a [Map for Blazor](slug://components/map/layers), you might want to customize the appearance of your map markers to differentiate them or match your application's theme. This knowledge base article also answers the following questions:
+
+- How can I set the color of a map marker in Blazor?
+- How to change the color of specific map markers based on their titles?
+- How to apply custom styles to map markers in a Blazor application?
+
+## Solution
+
+To customize the color of map markers in a Blazor application, you can [override the default theme styles](slug://themes-override).
+
+### Change the Color to All Map Markers
+
+To change the color for all markers use the following CSS approach:
+
+<div class="skip-repl"></div>
+````RAZOR
+<TelerikMap Class="my-map">
+    <!-- Map configuration -->
+</TelerikMap>
+
+<style>
+    .my-map.k-map .k-marker {
+        color: blue;
+    }
+</style>
+````
+
+### Customizing Specific Markers
```

Comment:
The grammar form of the verb should be the same in both headings (Change vs Customizing)